R/query.R

```diff
@@ -46,12 +47,13 @@ query.default <- function(connection, type, dataset, query, ...) {
 #' @export
 query.data.world <- function(connection, type = "sql", dataset, query, ...) {
```

Comment:
I know that @bryonjacob wanted to add support for parameterized queries. If it's a minor change, perhaps we should just add that too?